tests/EvmAcceptanceTests/test/Transfer.ts

```diff
@@ -153,20 +152,25 @@ describe("Transfer ethers", function () {
   it("should return check gas and funds consistency", async function () {
     let rndAccount = ethers.Wallet.createRandom();
 
-    const FUND = BigNumber.from(100_000_000_000);
+    const FUND = BigNumber.from(200_000_000_000_000_000n);
```

Comment:
Maybe using something like `ethers.utils.parseUnits("10", "ether")` is more readable.